us-banks/README.md

```diff
@@ -130,30 +131,29 @@ The reconciliation service configuration is in the [config](config) folder
 2. Start the reconciliation service docker `docker-compose -f docker-compose.yml up -d`
 3. Access GraphDB at `localhost:7400` 
 4. Load all the RDF data from [data/kb/ttl](data/kb/ttl). 
-5. Create the elasticsearch connectors by executing the two sparql queries [locations.sparql](config/reconciliator/locations.sparql) and [place.sparql](config/reconciliator/place.sparql)
+5. Create the elasticsearch connectors by executing the two sparql queries [locations.sparql](config/reconciliator/landmarks.sparql) and [place.sparql](config/reconciliator/places.sparql)
```

Comment:
The link name should probably be `[landmarks.sparql](...)`, not `[locations.sparql](...)`.